interview-questions.md

```diff
@@ -35,7 +35,7 @@ Then, research the question to expand on your answer. Even if you feel you have
   Your answer: A function is telling a computer how to do something.A method is deals with objects in the function.
 
 
-  Researched answer: The difference is that a method is associated with an object, while a function is not.
+  Researched answer: The difference is that a method is associated with an object, while a function is not.methods ar functions stored as object properties. A function takes on a task,takes in value,takes in an input and returns an output.
```

Comment:
During research try to find something beyond the definition.  Maybe a unique way that the topic is being used.